includes/class-event.php

```diff
@@ -131,11 +131,11 @@ public function save() {
 		}
 
 		if ( $this->exists() ) {
-			$success = Events_Store::_update_event( $this->id, $row_data );
+			$success = Events_Store::instance()->_update_event( $this->id, $row_data );
```

Comment:
This is great. One more step we could do is make $event_store a field in this class.

Then in the constructor you would do:
`this->event_store = Event_store::instance()`

And here just `this->event_store->_update_event`. 

The benefit of this is that then it is truly ready for unit testing.

In your unit test you could create an instance of Event and change the event_store to mocked one while testing these functions.

I dont think we need to rewrite this and the tests right now... but I am happy to help you out with that if you think it sounds good.

Reply:
Yep! Actually had something like this in the first iteration, but decided to strip it down since it kind of felt like extra code just for the sake of more code.

If/when we reach the point where we'd like to allow the store to be extensible/substitutable, or mock accordingly, then what you described can definitely be added - and in a way that won't require any BC breaks. So mainly for that reason I decided to punt it down the road for now, vs added code that maybe will, maybe won't, be used.